builder/builder.go

```diff
@@ -639,6 +639,157 @@ func (b *Builder) BuildUpdate(prefixflag string, minVersion int, format string,
 	return nil
 }
 
+func (b *Builder) buildUpdateWithNewSwupd(timer *stopWatch, mixVersion uint32, minVersion uint32, format uint32, skipSigning bool) error {
+	var err error
+
+	outputDir := filepath.Join(b.Statedir, "www")
+	if _, err = os.Stat(outputDir); err != nil {
+		return errors.Wrapf(err, "couldn't access the swupd output dir (www)")
```

Comment:
This isn't strictly necessary since `CreateManifests` makes the `outputDir` if it doesn't exist since it might not exist for the very first update built ever.

Reply:
Makes sense. Removed the check.